demos/react/froala3/README.md

```diff
@@ -80,14 +80,13 @@ You will also see any lint errors in the console.
         htmlAllowedTags: ['.*'],
         htmlAllowedAttrs: ['.*'],
         htmlAllowedEmptyTags: ['mprescripts'],
-        imageResize : false,
-        key: 'CA5D-16E3A2E3G1I4A8B8A9B1D2rxycF-7b1C3vyz==',
-        heightMax: 310,
-        useClasses: false
+        // imageResize : false,
+        // heightMax: 310,
+        // useClasses: false
```

Comment:
Since these are instructions inside a README file, I would refrain to include commented lines.

Reply:
Ok